src/utils/fileSystem.ts

```diff
@@ -759,6 +759,7 @@ export async function migrateFromLegacyExtension() {
         "diagnostics.extensionName",
         "diagnostics.metricKeys",
         "diagnostics.cardKeys",
+        "diagnostics.variables",
```

Comment:
This is not needed.
Since the legacy (<1.2.4) extension was not aware of this setting there's not need to make it aware now.